app/src/main/java/com/concordia/mcga/adapters/BuildingInformationArrayAdapter.java

```diff
@@ -0,0 +1,90 @@
+package com.concordia.mcga.adapters;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ArrayAdapter;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.concordia.mcga.activities.R;
+
+import java.util.ArrayList;
+
+
+
+/**
+ * Created by Charmander on 3/5/2017.
+ */
+
+public class BuildingInformationArrayAdapter extends ArrayAdapter<String> {
```

Comment:
He continued his work from the demo branch, even though we talked about not using it